### PR TITLE
The eighth large cleanup of JP filter

### PR DESCRIPTION
**Pull request by Yuki2718**
```
! ---------- Problematic rule
! https://github.com/easylist/easylist/issues/6009#issuecomment-685815738
/popIn.js

@@||lacoste.com/on/demandware.static/*/js/dist-refit/framework/base/popin.js
@@||focal.com/sites/*/themes/focal/js/popin.js
@@||lprs*.fr/assets/js/app/modules/popin.js$domain=leparisien.fr


! ---------- 0 usage in AG statistics and 0 hit in publicwww (depth:all)
/ads/techcrunchjapan_
/rikitake_banner
/top/head_s_bnr
_ds_adtag_


! ---------- Obsolete rules
||adcommerce.jp^$third-party
||adonavi.jp^$third-party
||ads.meropar.jp^$third-party
||bandoo.com^$image,~image,popup
||candy.smv.to^$third-party
||mobalyzer.net^$third-party
||q-sht-zidjk.co^$third-party
||rationalan.pro^$third-party
||trax-ad.jp^$third-party

@@||unrestern.info^
@@||wpteq.org/wp-content/plugins/wp-adblock-dedect/js/dedect.js
!+ PLATFORM(ios, ext_android_cb, ext_safari)
@@||wpteq.org^$generichide

||1dy2.com/js/zq.js
||coinch.me/js/link.js
||jartic.or.jp/guide/banner/
||javhub.net/highporn.html
||javhub.net/img/r18_
||prt.iza.ne.jp/common/images/v1/basic/banner_sankeiPhoto.png
||rationalan.pro^$domain=avgle.com
||ura.one/ad/link.js
||uratube.com/js/ad^
||y-bbs.net/k/

! Script URLs are alive but doesn't matter when the site itself is dead.
||gum.xbooks.to/js/pc/pc_2.js
||gum.xbooks.to/js/pc/pc_tsuibi*.js
||gum.xbooks.to/js/pc/pc_main_900_250.js
xbooks.to###ad_tsuibi
xbooks.to###ad_tsuibi2
xbooks.to##.movie-in-ad
xbooks.to###wrap > div[style="width:900px; height:250px; margin: -15px auto 15px auto;"]
xbooks.to###ad_interstitial_wrap
xbooks.to##.ad_overlay
||gum.xbooks.to/js/sp/sp_interstitial_move4.js
||gum.xbooks.to/js/sp/access_overlay_avt_03.js
||gum.xbooks.to/js/sp/sp_interstitial.js
||gum.xbooks.to/js/sp^

||sp.hazardlab.jp/imgs/banner/
sp.hazardlab.jp##div[id^="SP-NEWS_"]
hazardlab.jp##div[id*="banner_"]

69re.tv###wrapper > div.container > div[style="width: 100%"] > a[target="_blank"] > img
asagei.com##.link-asagei-h > a[target="_blank"] > img
crediblecar.life##a[rel="nofollow"] > img[style]
localplace.jp###bannerList
melos.media##.x1_ad
smv.to##.pc_content_tt_adsterra
smv.to##div[style^="width:900px;"][style*="height: 250px;"]
smv.to##.ad_movieupdown
sojson.com##div[class^="pl0 pr0 banner"]
sp-mobile-blog.com###text-136
sp-mobile-blog.com###text-149
sp-mobile-blog.com###text-156
webcartop.jp##.gourment-banner

limo.media#?#.right-column > div.right-column-block:has(> div[id^="outbrain_widget_"])


@@||jp.popin.cc/popin_discovery/$domain=livedoor.com
@@||ov.yahoo.co.jp^$script,domain=jp.msn.com


! ---------- Redundant rules
||js.blozoo.info/js/rsstool/blogparts.js
||widget-view.dmm.co.jp^$domain=mhometheater.com


! ---------- New rules
! For DNS: https://limo.media/
||adpicker.net^

! https://melos.media/training/64600/
||anymind360.com^

! http://yaraon-blog.com/
||google-analyticals.com^
yaraon-blog.com###single_content_sidebar

||jartic.or.jp/d/banner/substitute/$subdocument

! http://www.eroppu.com/
||mhub.work^

asagei.com##.ad-middle-wrap

! https://limo.media/articles/-/18423
limo.media###_popIn_recommend_text

beethovenmaster.com,global.sp.land.to##.SponsoredLink
sp-mobile-blog.com###text-100


! ---------- New extended rules
iza.ne.jp#?#.sideFixedWrapper > div[class^=wrapper-]:has([class^=ad-])


! ---------- New rules for Mobile
beethovenmaster.com##.ads


! ---------- New rules for Social
||beethovenmaster.com/bookmark/
beethovenmaster.com###bm02, .blockContent02

! https://localplace.jp/shika/ikebukuro/
localplace.jp##.matome-sns-btn
! https://localplace.jp/t100170065/
localplace.jp##.snsShare


! ---------- 0 usage in AG statistics, should be added to NOT_OPTIMIZED
! why image,~image?
||travian.jp^$image,~image,popup

! shinkotonami-gc.com: living domain
||203.183.90.6/$third-party
```

I didn't do anything about the last part so please take action on your end, there are some other weird $image,~image rules too.

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -35,6 +35,9 @@ realfilmizle1.com,bilimhaber.net,balikesirhaber.net,gruppal.com,somaaktuel.com,f
...
 !
+||beethovenmaster.com/bookmark/
+beethovenmaster.com###bm02, .blockContent02
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:39]: better do not add few selectors in the one rule

File: JapaneseFilter/sections/adservers.txt [hunk 2]
```diff
@@ -64,6 +63,10 @@
 ! Eimg.jp
 ||adingo.jp.eimg.jp^
 !
+||mhub.work^
+||adpicker.net^
+||anymind360.com^
+||google-analyticals.com^
```
Comment (Alex-302) [inline JapaneseFilter/sections/adservers.txt:69]: If used on another sites better add `$third-party`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/adservers.txt (hunk 2 above)
Comment (Alex-302) [reply JapaneseFilter/sections/adservers.txt:69]: Changed directly.

**Review by Alex-302 — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: Okay, will split to two filters.

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt]: Thx!

**Pull request merged**